resources/js/components/BetaPlotsView.vue

```diff
@@ -435,6 +466,30 @@
       closeImportForm() {
         this.importing = false
       },
+
+      async toggleReminders() {
+        try {
+          if (this.User.owns(this.site)) {
+            const {data} = await axios.put(`/web/sites/${this.site.id}/toggle-reminders`)
+            this.site.sends_reminders = data.sends_reminders
+            this.sendReminders = data.sends_reminders
+          } else {
+            const {data} = await axios.put(`/web/user-sites/${this.site.id}/toggle-reminders`)
+            this.userSite.sends_reminders = data.sends_reminders
```

Comment:
Updating objects and arrays in vue don't get reflected unless you update the parent object/array.

```suggestion
            this.userSite= {
                ...this.userSite,
                sends_reminders: data.sends_reminders
            }
```